packages/common/base/src/utils/apiKey/validateAPIKeyPrefix.test.ts

```diff
@@ -0,0 +1,115 @@
+import { describe, expect, test } from "vitest";
+
+import { validateAPIKeyPrefix } from "./validateAPIKeyPrefix";
+
+describe("validateAPIKeyPrefix", () => {
+    test("Should disallow API keys with old format", () => {
```

Comment:
normally I'd make a comment to split this up as:
```
context("when using an API key with old format", () => {
  const prefixes = ...
  
  it("throws an error", () => { ...}
```
but I don't know that we've added `context` to this repo